apps/prairielearn/src/pages/studentAssessment/studentAssessment.html.ts

```diff
@@ -0,0 +1,118 @@
+          ...resLocals,
+          navPage: '',
+        })}
+        <main id="content" class="container">
+          <div class="card mb-4">
+            <div class="card-header bg-primary text-white">
+              ${assessment_set.abbreviation}${assessment.number}: ${assessment.title}
+              ${assessment.group_work ? html` <i class="fas fa-users"></i> ` : ''}
+            </div>
+
+            <div class="card-body">
+              ${authz_result.mode === 'Exam' || authz_result.password != null
+                ? html`
+                    <p class="lead text-center">
+                      Please wait until instructed to start by a proctor
+                    </p>
+                  `
+                : ''}
+
+              <p class="lead text-center">
+                This is
+                <strong>${assessment_set.name} ${assessment.number}: ${assessment.title}</strong>
+                for <strong>${course.short_name}</strong>
+              </p>
+
+              ${authz_result.time_limit_min != null
+                ? html`
+                    <p class="lead text-center">
+                      The time limit for this assessment is
+                      <strong>
+                        ${authz_result.time_limit_min}
+                        ${authz_result.time_limit_min === 1 ? 'minute' : 'minutes'}
+                      </strong>
+                    </p>
+                  `
+                : ''}
+              ${assessment.group_work
+                ? renderEjs(
+                    import.meta.url,
+                    "<%- include('../partials/studentGroupControls.ejs'); %>",
+                    resLocals,
+                  )
+                : html`
+                    ${assessment.require_honor_code
+                      ? html`
+                          <div class="card card-secondary mb-4 test-class-honor-code">
+                            <ul class="list-group list-group-flush">
+                              <li class="list-group-item py-2">
+                                I certify that I am ${user.name} and I am allowed to take this
+                                assessment.
+                              </li>
+                              <li class="list-group-item py-2">
+                                I pledge on my honor that I will not give or receive any
+                                unauthorized assistance on this assessment and that all work will be
+                                my own.
+                              </li>
+                            </ul>
+
+                            <div class="card-footer text-center border-top-0 py-2">
+                              <span class="form-check d-inline">
+                                <input
+                                  type="checkbox"
+                                  class="form-check-input"
+                                  id="certify-pledge"
+                                />
```

Comment:
Bikeshed: the checkbox is actually outside of the form, and the only validation done that the checkbox is checked is done client-side, using JS to disable the button if the input is not checked. Someone could technically start the assessment without ever checking this box by re-enabling the button using JS. Is this a concern? Should there be some level of validation that the box has actually been selected?

Reply:
What is this hypothetical student going to do if they're caught cheating, say "I am very clever and started this assessment without checking the box, so you can't prove I knew I had to maintain academic integrity"? 😂